--- board-legend.orig	2010-08-19 21:13:19.346195152 +0200
+++ arch/arm/mach-msm/board-legend.c	2010-08-19 21:12:21.407195151 +0200
@@ -176,7 +176,7 @@
 	{
 		.name = "light_sensor",
 		.category = MICROP_FUNCTION_LSENSOR,
-		.levels = { 0, 0x21, 0x4D, 0xDC, 0x134, 0x18D, 0x1E5, 0x3FF, 0x3FF, 0x3FF },
+		.levels = { 0, 0x21, 0x4D, 0xDC, 0x134, 0x18D, 0x1E5, 0x2BA, 0x35C, 0x3FF },
 		.channel = 3,
 		.int_pin = 1 << 9,
 		.golden_adc = 0xC0,
